### ISSUE TITLE
T-values data from Brain_Data.regress() are returned as matrix type

### ISSUE THREAD

**Issue by ejolly** [ljchang/nltools#170]
Not sure why but t-values should be consistent with others, e.g.:

```
x = Brain_Data(...)
out = x.regress()
type(out['beta'].data) # np.ndarray
type(out['df'].data) # np.ndarray
type(out['residual'].data) # np.ndarray
type(out['t'].data) # np.matrixlib.defmatrix.matrix
```

**Comment by ljchang** [comment]
makes sense to output them as Brain_Data() type

**Comment by ejolly** [comment]
Resolved in #149
